ClosedXML_Examples/Misc/WorkbookProtection.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using ClosedXML.Excel;
+
+namespace ClosedXML_Examples.Misc
+{
+    public class WorkbookProtection : IXLExample
+    {
+        #region Variables
```

Comment:
Remove the empty regions, please. I know many examples have them. No idea why... it's just fluff.